business_requirement_deliverable_cost/readme/INTRODUCTION.rst

```diff
@@ -0,0 +1,10 @@
+This module improves the cost control of the original module with the following:
```

Comment:
BTW: there is no INTRODUCTION file in fragment.
I will rework the README later on